### PR TITLE
[OKTA-735570]: Replaced non inclusive terms in a few files

### PR DESCRIPTION
**Pull request by ishan-krishna-okta**
## Description:
- **What's changed?** Replaced a few non-inclusive terms with inclusive ones
- **Is this PR related to a Monolith release?** 

### Resolves:

* [OKTA-735570](https://oktainc.atlassian.net/browse/OKTA-735570)

### CODE REVIEW

File: packages/@okta/vuepress-site/books/api-security/sanitizing/sanitize-inputs/index.md [hunk 1]
```diff
@@ -6,13 +6,13 @@ title: Sanitize Inputs - Sanitizing Data
 
 ## Sanitize Inputs {#sanitize-inputs}
 
-Sanitizing inputs can be a good option when the input format is not strict but still somewhat predictable, such as phone numbers or other free-text fields. There are a few different ways to sanitize inputs, you could use a whitelist, a blacklist, or escape input.
+Sanitizing inputs can be a good option when the input format is not strict but still somewhat predictable, such as phone numbers or other free-text fields. There are a few different ways to sanitize inputs, you could use a allowlist, a blocklist, or escape input.
 
-### Sanitize Input Using a Whitelist
-When sanitizing data with a whitelist, only valid characters/strings matching a given pattern are kept.  For example, when validating a phone number there are multiple formats people use, US phone numbers could be written as `555-123-1245`, `(555) 123-1245`, `555.123.1245`, or a similar combination. Running any of these through a whitelist that only allows numeric characters would leave `5551231245`.
+### Sanitize Input Using a Allowlist
```
Comment (emikhasyak-okta) [inline packages/@okta/vuepress-site/books/api-security/sanitizing/sanitize-inputs/index.md:11]: Should this be "an Allowlist"?
cc: @damianali-okta to confirm

**Review by emikhasyak-okta — COMMENTED** [review]
The change looks pretty straightforward. Have one nit regarding article

File: packages/@okta/vuepress-site/books/api-security/sanitizing/sanitize-inputs/index.md (hunk 1 above)
Comment (ishan-krishna-okta) [reply packages/@okta/vuepress-site/books/api-security/sanitizing/sanitize-inputs/index.md]: Good catch.. Updated!

**Review by emikhasyak-okta — APPROVED** [review]
🚢

**Pull request merged**